packages/node_modules/@ciscospark/helper-image/src/process-image.js

```diff
@@ -12,9 +12,11 @@ import {pick} from 'lodash';
  * @param {Number} options.thumbnailMaxWidth
  * @param {Number} options.thumbnailMaxHeight
  * @param {Boolean} options.enableThumbnails
+ * @param {Object} options.logger
  * @returns {Promise<Array>} Buffer, Dimensions, thumbnailDimensions
  */
-export default function processImage({
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Any reason why the default was removed? [Our reasoning](https://github.com/ciscospark/web-styleguide/tree/master/javascript#modules--prefer-default-export) behind the rule.

Reply:
process-image.browser.js has two functions detectFileType() and processImage() which are called from outside the helper-image package and hence the default is removed. The default is also removed from process-image.js to keep both the node and browser version in sync.

Reply:
Sounds good. Could you please make a note in comments above the disable for that? I think we are good once those changes are in place.